bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusBinding.java

```diff
@@ -222,7 +239,10 @@ protected void internalUpdateItem(String slaveName, InputRegister[] registers, S
                 double tmpValue = ((DecimalType) newState).doubleValue() * rawDataMultiplier;
                 newState = new DecimalType(String.valueOf(tmpValue));
             }
-
+            if (config.getItemClass() == RollershutterItem.class) {
```

Comment:
Sorry, I misread the number handling. This is slightly different case here so this is OK.